.taskcluster.yml

```diff
@@ -123,7 +123,7 @@ tasks:
       - "secrets:get:project/focus/tokens"
       - "queue:route:index.project.mobile.focus.release.latest"
     payload:
-      maxRunTime: 3600
+      maxRunTime: 7200
```

Comment:
This is the task for releases. You want to update the `maxRunTime` above to make the PR task run longer.